src/js/types/types.ts

```diff
@@ -0,0 +1,15 @@
+export type Degrees = number
+
+export type Feet = number
+
+export type Knots = number
+export type NauticalMiles = number
+export type Minutes = number
+export type KiloHertz = number
+export type MegaHertz = number
+export type Metres = number

```

Comment:
Perhaps these should be in a `units.ts` file?

Reply:
types.ts probably isnt a very good name yeah, but id rather keep it with Coordinates for simplicity, how about math.ts so its like the rust?

Reply:
Sure! Works for me 👍